code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1630,3 +1630,14 @@
 	description = "blue sparkles that get everywhere"
 	color = "#4040FF" //A blueish color
 	glitter_type = /obj/effect/decal/cleanable/glitter/blue
+
+/datum/reagent/pukonium
```

Comment:
Having a chemical that increases disgust seems silly since some races(Slime people for example) LIKE toxic food. A reagnent designed to disgust all races seems better to me.